docs/src/content/docs/de/components/badges.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: Abzeichen
+description: Lerne, wie du in Starlight Abzeichen verwenden kannst, um zusätzliche Informationen anzuzeigen.
+---
+
+import { Badge } from '@astrojs/starlight/components';
+
+Um wenige Informationen, wie z.B. einen Status oder eine Kategorie, anzuzeigen, verwende die Komponente `<Badge>`.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<Badge slot="preview" text="Neu" />
+
+</Preview>
+
+## Import
+
+```tsx
+import { Badge } from '@astrojs/starlight/components';
+```
+
+## Verwendung
+
+Zeige ein Abzeichen mit der Komponente `<Badge>` an und übergib den Inhalt, den du anzeigen möchtest, an das Attribut [`text`](#text) der Komponente `<Badge>`.
+
+Standardmäßig wird für das Abzeichen die Akzentfarbe des Aussehens deiner Website verwendet.
+Um eine eingebaute Abzeichen-Farbe zu verwenden, setze das Attribut [`variant`](#variant) auf einen der unterstützten Werte.
+
+<Preview>
+
+```mdx
+import { Badge } from '@astrojs/starlight/components';
+
+<Badge text="Hinweis" variant="note" />
+<Badge text="Erfolg" variant="success" />
+<Badge text="Tipp" variant="tip" />
+<Badge text="Warnung" variant="caution" />
+<Badge text="Gefahr" variant="danger" />
+```
+
+<Fragment slot="markdoc">
+
+```markdoc
+{% badge text="Hinweis" variant="note" /%}
+{% badge text="Erfolg" variant="success" /%}
+{% badge text="Tipp" variant="tip" /%}
+{% badge text="Warnung" variant="caution" /%}
+{% badge text="Gefahr" variant="danger" /%}
+```
+
+</Fragment>
+
+<Fragment slot="preview">
+	<Badge text="Hinweis" variant="note" />
+	<Badge text="Erfolg" variant="success" />
+	<Badge text="Tipp" variant="tip" />
+	<Badge text="Warnung" variant="caution" />
+	<Badge text="Gefahr" variant="danger" />
+</Fragment>
+
+</Preview>
+
+### Verschiedene Größen verwenden
+
+Verwende das Attribut [`size`](#size), um die Größe des Abzeichentextes zu steuern.
+
+<Preview>
+
+```mdx /size="\w+"/
+import { Badge } from '@astrojs/starlight/components';
+
+<Badge text="Neu" size="small" />
+<Badge text="Neu und verbessert" size="medium" />
+<Badge text="Neu, verbessert und größer" size="large" />
+```
+
+<Fragment slot="markdoc">
+
+```markdoc /size="\w+"/
+{% badge text="Neu" size="small" /%}
+{% badge text="Neu und verbessert" size="medium" /%}
+{% badge text="Neu, verbessert und größer" size="large" /%}
+```
+
+</Fragment>
+
+<Fragment slot="preview">
+	<Badge text="Neu" size="small" />
+	<Badge text="Neu und verbessert" size="medium" />
+	<Badge text="Neu, verbessert und größer" size="large" />
+</Fragment>
+
+</Preview>
+
+### Abzeichen anpassen
+
+Passe Abzeichen an, indem du andere `<span>`-Attribute wie `class` oder `style` mit benutzerdefiniertem CSS verwendest.
+
+<Preview>
+
+```mdx "style={{ fontStyle: 'italic' }}"
+import { Badge } from '@astrojs/starlight/components';
+
+<Badge text="Individuell" variant="success" style={{ fontStyle: 'italic' }} />
```

Comment:
Just to double check, and this is probably my translation tool that is not good enough, is "Individuell" the expected word for "Custom"?

Reply:
There are probably many different words possible for the german translation of "custom", "individuell" is probably NOT one of them xD. Nonetheless, I think it fits the situation perfectly, but I would love to hear thoughts from other Native Speakers, f.e. @delucis